modules/apps/application-list/application-list-test/src/testIntegration/resources/com/liferay/application/list/deploy/hot/test/dependencies/control-panel-entry-liferay-portlet.xml

```diff
@@ -2,7 +2,7 @@
 
 <liferay-portlet-app>
 	<portlet>
-		<portlet-name>1</portlet-name>
+		<portlet-name>90</portlet-name>
```

Comment:
@W2jx Did you try to put here a custom portlet name, and in the test, deploy a test/dummy portlet with that name?

`90` is a very special portlet which means "PORTAL" itself. We should avoid using it.

Reply:
I tried to deploy a test portlet, but it didn't work. I can try again